Mathlib/Data/Finset/Lattice/Fold.lean

```diff
@@ -783,6 +791,11 @@ theorem sup'_product_right {t : Finset γ} (h : (s ×ˢ t).Nonempty) (f : β ×
     (s ×ˢ t).sup' h f = t.sup' h.snd fun i' => s.sup' h.fst fun i => f ⟨i, i'⟩ := by
   rw [sup'_product_left, Finset.sup'_comm]
 
+/-- Analog of `Finset.isLUB_sup` requiring a nonempty set. -/
```

Comment:
I think we should just use `set_option linter.docPrime false in` here, since the lemmas is about a primed definition.

Reply:
Good suggestion. I didn't know how to disable it, but I agree no docstring is really necessary.